clang-tools-extra/docs/clang-tidy/checks/readability/math-missing-parentheses.rst

```diff
@@ -0,0 +1,20 @@
+.. title:: clang-tidy - readability-math-missing-parentheses
+
+readability-math-missing-parentheses
+====================================
+
+Check for mising parantheses in mathematical expressions that involve operators
+of different priorities.
+
```

Comment:
Consider adding some longer description, like rationale, for example:

_Parentheses in mathematical expressions clarify the order of operations, especially with different-priority operators. Lengthy or multiline expressions can obscure this order, leading to coding errors. IDEs can aid clarity by highlighting parentheses. Explicitly using parentheses also clarify what the developer had in mind when writing the expression. Ensuring their presence reduces ambiguity and errors, promoting clearer and more maintainable code._

Reply:
done